icons/rxdb.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xml:space="preserve" xmlns="http://www.w3.org/2000/svg"><title>RxDb icon</title><path d="M4.2 15.6v4.9h6.4v1.2h1.2v-1.2h1.1v2.3h1.2v-2.3h1.1v2.3h1.2V24h1.1v-2.3h1.2v1.1h1.1v-7.2z M4.2 14.5h15.6v-5H4.2v5z M6.5 0v2.3H5.3V1.2H4.2v7.2h15.6V3.5h-6.4V2.3h-1.2v1.2h-1.1V1.2H9.9v2.3H8.8V1.2H7.6V0z"/></svg>
```

Comment:
I'm seeing a [few differences](https://user-images.githubusercontent.com/1830867/117814264-fbba7100-b25b-11eb-98ad-c22dc83f9490.png) between your version in red, and the source icon in black. Would you mind taking another look at this, and don't forget to update the title to 'RxDB' to match the JSON!

Reply:
OK - thats updated. I'm not sure what tool you we using to do the diffs between the two versions but the `.svg` file does have updated path nodes